packages/grafana-ui/src/components/Segment/SegmentAsync.tsx

```diff
@@ -28,12 +30,25 @@ export function SegmentAsync<T>({
 
   if (!expanded) {
     const label = _.isObject(value) ? value.label : value;
+    const styles = getStyles();
+
     return (
       <Label
         onClick={fetchOptions}
+        disabled={disabled}
         Component={
           Component || (
-            <a className={cx('gf-form-label', 'query-part', !value && placeholder && 'query-placeholder', className)}>
+            <a
```

Comment:
Using `a` without href is a bad accessibility practice, plus it seems like we're not using it as a link here at all, so can just change it to `span` and apply existing styles.